docs/source/roles/system_manager/manage_data.md

```diff
@@ -34,6 +34,28 @@ The following steps show how to generate a temporary write-only upload token tha
 - The data provider should now be able to upload data by following {ref}`these instructions <process_data_ingress>`
 - You can validate successful data ingress by logging into the SRD for the SRE and checking the `/data` volume, where you should be able to view the data that the data provider has uploaded
 
+(roles_system_manager_multiple_providers)=
+
+## Multiple data providers
+
+In some projects, there may be more than one data provider responsible for uploading data. Two potential issues that may occur are _name clashes_ and _data leakage_.
+
+If all data providers are uploading to the same storage container, then name clashes may occur. There is no protection against overwriting files during upload.
+Thus, if that different data providers upload files with the same name, then the earlier upload will be overwritten.
+This can be avoided by providing each data provider with their own subfolder on the storage container.
```

Comment:
```suggestion
This can be avoided by providing each data provider with their own subfolder on the storage container and ensuring that each uploads only to their subfolder.
```